### PR TITLE
Add Course Comparator Project

### PR DESCRIPTION
**Pull request by 108anup**

### CODE REVIEW

File: Pull Request [hunk 1]
```diff
@@ -0,0 +1,6 @@
+I don't know how to make pull request from a repository that has not been forked from the hackathon repository.
+Please guide me.
```
Comment (swapagarwal) [inline Pull Request]: You've already forked it.
Now edit the README file and replace the "Sample Project" placeholder with your project's details.

**Comment by 108anup** [comment]
I have edited the Readme.md file

**Comment by swapagarwal** [comment]
Cool, but there are 3 commits and I'd prefer squashing (google is your friend) this into a single commit. Let me know if you need any help.

**Comment by 108anup** [comment]
Done, Confirm the changes

**Comment by swapagarwal** [comment]
Great!
I just noticed you changed the date as well. That was the hackathon date!

**Comment by 108anup** [comment]
ok changed the date, is it fine now?

**Comment by swapagarwal** [comment]
Nice work @108anup :+1:

**Pull request merged**